Neos.Media/Classes/Domain/Model/AdjustmentCapableInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace Neos\Media\Domain\Model;
+
+use Neos\Media\Domain\Model\Adjustment\ImageAdjustmentInterface;
+
+interface AdjustmentCapableInterface
+{
+    public function getAdjustments(): \IteratorAggregate;
```

Comment:
Imo this is a better fit ;)

```suggestion
    /** @return iterable<ImageAdjustmentInterface> */
    public function getAdjustments(): iterable;
```